drivers/mrf24j40/include/mrf24j40_registers.h

```diff
@@ -40,7 +40,7 @@ extern "C" {
 
 /**
  * @brief FIFO-Address-Map
- ****@{
+ *****@{
```

Comment:
why so many asterisks?

Reply:
I think uncrustify did that, let me check that.

Reply:
The missing space before the `@{` caused some alignment issues with uncrustify I think. Is fixed now.